### PR TITLE
[ADD] stock_block_lot

### PR DESCRIPTION
**Pull request by clonedagain**
Module to block serial numbers/lots without installing MRP. Split from mrp_lock_lot

As discussed in https://github.com/OCA/stock-logistics-workflow/issues/110#issuecomment-102438140.

### CODE REVIEW

**Comment by clonedagain** [comment]
I'll look into the problem Travis noticed

**Comment by clonedagain** [comment]
@pedrobaeza I don't think I can do that because I found out mrp_lock_lot actually depends on mrp for `wizard/mrp_product_produce_view.xml`, which I had not seen at first glance.

**Comment by pedrobaeza** [comment]
Uhm, I see. Then keep this wizard in mrp_lock_lot and put the rest on the new module.

**Comment by clonedagain** [comment]
Yes that's what I tried to do, did I miss something?

File: stock_lock_lot/wizard/mrp_product_produce_view.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pedrobaeza) [inline stock_lock_lot/wizard/mrp_product_produce_view.xml]: This XML should be in mrp_lock_lot

**Comment by clonedagain** [comment]
OK I had wrongly duplicated the file, it should be fixed now.

**Comment by clonedagain** [comment]
I had wrongly modified `stock_quant_manual_assign_without_lock_lot/__openerp__.py` too, I've just fixed it.

File: mrp_lock_lot/__openerp__.py [hunk 2]
```diff
@@ -34,18 +35,12 @@
...
     "data": [
-        "data/mrp_lock_lot_data.xml",
-        "wizard/stock_transfer_details_view.xml",
         "wizard/mrp_product_produce_view.xml",
-        "wizard/wiz_lock_lot_view.xml",
-        "views/product_category_view.xml",
-        "views/stock_production_lot_view.xml",
-        "views/stock_quant_view.xml",
     ],
+    "auto": True,
```
Comment (pedrobaeza) [inline mrp_lock_lot/__openerp__.py]: Correct key is auto_install, but I don't think this module must be auto-installed.

Comment (clonedagain) [inline mrp_lock_lot/__openerp__.py]: Don't you think is should?  all it contains now is the mrp-specific view to prevent adding blocked lots.

Comment (pedrobaeza) [inline mrp_lock_lot/__openerp__.py]: You're right, this utility can come as auto_install. The main module don't.

**Comment by clonedagain** [comment]
Is it OK for you?

**Pull request closed without merging**